blog/2023-10-11-pause-resume-sessions/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: pin-sessions-landing-page
+title: |
+  RenkuLab Update: Pause and Resume Sessions
+authors: [laura]
+tags:
+  - renkulab update
+  - sessions
```

Comment:
```suggestion
  - Sessions
```

Reply:
it's a tag - should it be capitalized?

Reply:
Yes, see how it renders on the tags page: https://blog.renkulab.io/tags.